Scarb.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "openzeppelin"
+version = "0.1.0"
+cairo-version = "1.1.0"
+authors = ["OpenZeppelin"]
```

Comment:
```suggestion
authors = ["OpenZeppelin Community [maintainers@openzeppelin.org]"]
```

Reply:
can this be a single string instead of an array?

Reply:
Like in Cargo, the `authors` field has to be an array.